README.md

```diff
@@ -72,12 +74,12 @@ The below command will run Lit on the Bitcoin testnet3 network
 (Note: Windows users should take off `./` but need to change `lit` to `lit.exe`)
 
 ```bash
-./lit --tn3=true
+./lit --tn3 localhost
```

Comment:
Hi! I pushed a fix for something similar to https://github.com/mit-dci/lit/pull/444. I wanted to make sure that this works, but when I run this on my system, I get:

```
...
...
...
2019/01/22 22:11:13.949962 [INFO] created hardcoded genesis header at /Users/arunasank/.lit/testnet3/header.bin
2019/01/22 22:11:13.949992 [INFO] opened header file /Users/arunasank/.lit/testnet3/header.bin
2019/01/22 22:11:13.950003 [INFO] Attempting connection to node at localhost:18333
2019/01/22 22:11:13.952335 [ERROR]dial tcp4 127.0.0.1:18333: connect: connection refused
2019/01/22 22:11:13.952354 [ERROR] Tried to connect to all available node Addresses. Failed
2019/01/22 22:11:13.952359 [INFO] Couldn't dial node localhost, Moving on
2019/01/22 22:11:13.952365 [ERROR] Can't connect to host localhost
2019/01/22 22:11:13.952369 [ERROR] NewWallit Hook.Start crash  Couldn't establish connection with node. Exiting.
2019/01/22 22:11:13.953296 [INFO] adr 0 hash is <hash>
2019/01/22 22:11:13.953742 [ERROR]Unsupported coin daemon or coin daemon not running
2019/01/22 22:11:13.953756 [ERROR]Unsupported coin daemon or coin daemon not running
2019/01/22 22:11:13.953769 [DEBUG] Trying to listen on %!s(int=2448)
2019/01/22 22:11:13.953778 [DEBUG] eventbus: Published event: lnp2p.listen.start
2019/01/22 22:11:13.953784 [INFO]PORT: 2448
2019/01/22 22:11:13.953955 [INFO] Listening with ln address: <address>
2019/01/22 22:11:13.953985 [INFO] Done, tried 0 hosts
...
```